tests/test_image_handling.py

```diff
@@ -25,3 +41,32 @@ def test_scale_image(self):
         assert scale_image(self.img_96x85, False, "50,", "image/jpeg") == self.img_50x44
         assert scale_image(self.img_96x85, False, "60,44", "image/jpeg") == self.img_49x44
         assert scale_image(self.img_96x85, False, ",44", "image/jpeg") == self.img_49x44
+
+    #TODO: Specifiy exception
+    def test_parse_region_string(self):
+        with pytest.raises(Exception):
+            parse_region_string("x,y,w,h")
+            parse_region_string("")
+            parse_region_string(",,,")
```

Comment:
Met deze test je of één van deze drie een Exception geeft, niet of ze allemaal een exception geven.

Reply:
Goed punt. Ik wilde het meer leesbaar maken... oops

Reply:
Toegevoegd